index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="This is a boilerplate for building websites." />
+    <title>SPAcE DiNoSAuR</title>
+
+    <link rel="stylesheet" href="css/reset.css" type="text/css" />
+    <link rel="stylesheet" href="css/style.css" type="text/css" />
+  </head>
+
+  <body>
+    <div id="fadeImage">
+      <div id="container">
+       <header id="page-header">
+           <div class="clear-fix">
+              <div class="nav">
+                <ul>
+                  <li>
+                    <a href="about.html">ABOUT</a>
+                  </li>
+                  <li>
+                    <a href="music.html">MUSIC</a>
+                  </li>
+                  <li>
+                    <a href="GIGS.html">GIGS</a>
+                  </li>
+                  <li>
+                    <a href="updates.html">UPDATES</a>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </header>
+
+          <main>
+            <div class="content">
+              <h1>SPAcE DiNoSAuR</h1>
+              <img src="http://placehold.it/450x700">
+            </div>
+          </main>
+          <footer>
+            <div id="social">
+  						<a href="facebook.com"><img class="media" src="facebook.png"></a>
+  						<a href="twitter.com"><img class="media" src="twitter.png"></a>
+  					</div>
+          </footer>
+      </div>
+    </div>
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+  <script src="index.js"></script>
```

Comment:
Just like with CSS and images, try making a javascript folder so you keep things a bit more organized.